src/DonationForms/resources/registrars/templates/fields/Phone.tsx

```diff
@@ -0,0 +1,21 @@
+import InputMask from 'react-input-mask';
+
+import {PhoneProps} from '@givewp/forms/propTypes';
+
+export default function Phone({Label, ErrorMessage, fieldError, description, phoneFormat, inputProps}: PhoneProps) {
+    const FieldDescription = window.givewp.form.templates.layouts.fieldDescription;
+
+    return (
+        <label>
+            <Label />
+            {description && <FieldDescription description={description} />}
+            {phoneFormat === 'domestic' ? (
```

Comment:
Are we only supporting input masking for this `domestic` format?
Should this formatting be informed by the user client locale?

Reply:
Thats a great question. It would seem so. I was reading the NPM package details it looks like there is a known issue with browser's autofill that might be providing inconsistent behavior. Which may be the reasoning behind it. I actually did not get to speak to Paulo about this specific block.

Reply:
Ok - I'm not too worried about this right now and we can revisit this later. I don't think it needs to hold up this PR.

Reply:
Hey @kjohnson and @JoshuaHungDinh, I'd like to add something to this discussion. I've been following a basic rule for all cases here: if something is not specified in the designs (such as date formats), we'll continue to use it as it works in v2.